src/hausdorff.jl

```diff
@@ -121,41 +130,46 @@ hausdorff(imgA::GenericGrayImage, imgB::GenericGrayImage) =
 modified_hausdorff(imgA::GenericGrayImage, imgB::GenericGrayImage)  =
     ModifiedHausdorff()(imgA, imgB)
 
-# precalculate psets to accelerate computing
+# precalculate distance transforms to accelerate computing
 function pairwise(d::GenericHausdorff,
                   imgsA::AbstractVector{<:GenericGrayImage},
                   imgsB::AbstractVector{<:GenericGrayImage})
-    psetsA = [img2pset(imgA) for imgA in imgsA]
-    psetsB = [img2pset(imgB) for imgB in imgsB]
+    dtsA = hausdorff_transform.(imgsA)
+    dtsB = hausdorff_transform.(imgsB)
 
     m, n = length(imgsA), length(imgsB)
     D = zeros(m, n)
 
     for j=1:n
-      psetB = psetsB[j]
+      imgB = imgsB[j]
```

Comment:
similarily, will it be more efficient to use `imgB = @view imgsB[j]` and `imgA = @view imgsA[i]`?

Reply:
@johnnychen94 is `imgsB[j]` actually copied here? Won't `imgB` just hold a reference to `imgsB[j]`?

Reply:
I share your point of view @jsenn, let's wait for @johnnychen94 to confirm.

Reply:
Oops, you are correct. I didn't realize that `imgsB` is an array of arrays.